acceptance/tests/mesh-gateway/mesh_gateway_test.go

```diff
@@ -27,7 +27,8 @@ func TestMeshGatewayDefault(t *testing.T) {
 	secondaryContext := env.Context(t, environment.SecondaryContextName)
```

Comment:
Not required for this PR, but it'd be nice if we could rename this package to say that it's wan fed test rather than `mesh-gateway` because mesh gateways are used in many places now

Reply:
done! renamed to `wan-federation`